### PR TITLE
Feature/rm delete cover

### PR DESCRIPTION
**Pull request by mr-ryan12**
Hey! I added some functionality to remove a saved cover when the user double clicks on it. All conflicts have been resolved and it has been checked for functionality multiple times. Everything appears to work just fine, but please let me know if you find any bugs and I will do my best to resolve them.

### CODE REVIEW

File: src/main.js [hunk 1]
```diff
@@ -170,13 +172,23 @@ function saveCover() {
 }
 
 function displaySavedCovers() {
-  var savedCoversSectionHTML = ``;
+  savedCoversSection.innerHTML = ``;
```
Comment (GraceGardner) [inline src/main.js:175]: Nice, this re assignment really cleaned up this function.

File: src/main.js [hunk 2]
```diff
@@ -170,13 +172,23 @@ function saveCover() {
...
+
+function removeCover(event) {
+  var something = event.target.id;
+  for (var i = 0; i < savedCovers.length; i++) {
+    if (`${savedCovers[i].id}` === something) {
+      savedCovers.splice(i, 1);
+    }
+  }
+  displaySavedCovers();
+}
```
Comment (GraceGardner) [inline src/main.js:194]: I think git hub is encouraging us to add an extra line at the bottom that is just empty

**Review by GraceGardner — CHANGES_REQUESTED** [review]
This PR has some great new functionality. It's super clean and well organized. The only thing is that extra space at the end of the file that we need to add. Nice work! This is awesome!

File: src/main.js (hunk 2 above)
Comment (mr-ryan12) [reply src/main.js]: Thank you for catching that! It's been fixed and should be all good to go! :)

**Review by GraceGardner — APPROVED** [review]

**Pull request merged**